### DISCUSSION TITLE
FIEF with graphql Ariadne fast api API ?

### DISCUSSION THREAD

**Discussion by MathieuGrosso** [fief-dev/fief#339] [Ideas]
Will it be possible to integrate fief auth on fastapi call ? and if yes how to do it ? 

I have seen that there is already a thread for strawberry, so I guess it should  be possible, just wondering if you were thinking about integrating it in documentation. 
Thanks !

**Comment by frankie567** [comment]
Have you checked our FastAPI example: https://docs.fief.dev/integrate/python/fastapi/ ? It should work out-of-the-box, even if a GraphQL endpoint, nothing special about it.
